examples/conv_lstm.py

```diff
@@ -8,11 +8,18 @@
 from keras.layers.normalization import BatchNormalization
 import numpy as np
 import pylab as plt
+from keras import backend as K
 
 # We create a layer which take as input movies of shape
 # (n_frames, width, height, channels) and returns a movie
 # of identical shape.
 
+
+if K.backend() == 'cntk':
+    raise RuntimeError('CNTK could not run this example, due to the missing support for padding '
+                       'with non-specified input shape. To run it with CNTK, please update the '
+                       'input_shape in top COnvLSTM2D layer from (None, 40, 40, 1) to (15, 40, 40, 1)')
```

Comment:
Specify a shape and then check if the shape is okay. Otherwise, even if I set input_shape like yu say, the exception still throw.

input_shape=(15,40,40,1)
if K.backend()=='cntk' and input_shape[0] is None:
   raise...